style.css

```diff
@@ -0,0 +1,112 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap");
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #170606;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
[Optional - but highly recommended]
At the moment, the display of your app is centred horizontally and vertically, with a lot of redundant space:

Your display:
https://www.loom.com/share/74dfeadd52e948e8a5271e502564e8cf

My suggestion:
https://www.loom.com/share/488ec7addd2345dc921aa1db417daacc

- [ ] For this to work, you have to remove the code from lines 11 to 14 completely - you don't need them.